### ISSUE TITLE
Failed test 'All jobs executed 159/160'

### ISSUE THREAD

**Issue by eserte** [jmico/beekeeper#1]
On some of my smoker systems the test suite fails like this:
```
#   Failed test 'All jobs executed 159/160'
#   at /home/cpansand/.cpan/build/2021042314/Beekeeper-0.01-1/t/lib/Tests/Resend.pm line 108.
#   (in Tests::Resend->test_02_int)
#          got: '159'
#     expected: '160'
# Looks like you failed 1 test of 68.
t/07-resend.t ....... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/68 subtests 
```

**Comment by jmico** [comment]
I am aware of tests failing on several smoker platforms, and I am already working on fixes.

Thanks for reporting this, and thank you for your comprehensive testing, it has been very useful :)

**Comment by jmico** [comment]
Most tests had a very tight wait for asynchronous task completion, failing randomly when not enough CPU resources were available.
Fixed on version 0.02
